github/github.go

```diff
@@ -32,11 +33,12 @@ func New(baseUrl, acceptHeader, appID string, appPrivateKey []byte) GithubInterf
 }
 
 // A function that calls the github api, uses github base url
-// accepts a relative url that starts with "/" (ex: /app/installation)
+// accepts an endpoint that starts with "/" (ex: /app/installation)
+// accept query that will be set after "?" (ex: page=2&per_page=50)
 // sets the default "Accept" header to the one used by github apps
 // authToken is the full auth token not just the value (ex: authToken="Bearer {token}")
-func (g *github) CallGithub(endpoint, verb, authToken string) ([]byte, error) {
-	fullUrl := g.baseUrl + endpoint
+func (g *github) CallGithub(endpoint, verb, query, authToken string) ([]byte, error) {
+	fullUrl := fmt.Sprintf("%s/%s?%s", g.baseUrl, endpoint, query)
```

Comment:
I think there is a bug here

adding the `/` (`%s/%s`)
when we call this method we pass something like`/installation/repositories` this will make it look like `api.github.com//installation..`

Reply:
Try https://github.com//kintohub//utils-go//pull//10.
Should not be an issue no?

Reply:
Most web servers ignore them coz it happens often, but it's not recommended to have even if github ignores it

Reply:
OK :D
Let me change it hehe